test-requirements.txt

```diff
@@ -3,5 +3,5 @@ black
 boto3
 flake8
 pytest
-tensorflow==1.15
+tensorflow>=1.15.2
```

Comment:
Why is the patch version necessary?

Reply:
https://github.com/twosixlabs/armory/pull/142 
https://github.com/tensorflow/tensorflow/releases/tag/v1.15.2